src/renderer/common/js/utils.js

```diff
@@ -85,18 +85,60 @@
         ipcRenderer.removeAllListeners(messageChannel);
     };
 
-    /*
+    /**
+     * A custom function for handling activation of the "Quick Folders" QSS button.
+     * opens a provided url in the default browser using electron's shell
+     * @param {String} siteUrl - cloud folder's url
+     * @param {Boolean} alwaysUseChrome - true to use chrome, rather than the default browser.
+     * @param {Boolean} forceFullScreen - the function requires the browser to be open maximized
+     */
+    gpii.psp.openUrl = function (siteUrl, alwaysUseChrome, forceFullScreen) {
```

Comment:
Stick this in a Windows namespace to make it clear that we need to write a replacement for it once MacOS support starts to appear